src/msg/async/AsyncConnection.cc

```diff
@@ -868,7 +868,7 @@ ssize_t AsyncConnection::_process_connection()
         if (r < 0)
           goto fail;
 
-        center->create_file_event(cs.fd(), EVENT_READABLE, read_handler);
+        center->create_file_event(cs.fd(), EVENT_WRITABLE, read_handler);
```

Comment:
I got it,
On the one hand, whether we can through macro definition to solve the problem, on the other hand, is it possible to fd has Always can't readable?  use  "center->dispatch_event_external(read_handler)" is better at this moment?